indico/modules/attachments/client/js/legacy.js

```diff
@@ -184,10 +184,15 @@
       .on('click', '.js-dialog-action', function(e) {
         e.preventDefault();
         const $this = $(this);
+        let title = $this.data('title');
+        if (!title && title !== undefined) {
+         // if `data-title` is present without a value, fall back to the title attr
+         title = $this.attr('title') || $this.data('qtip-oldtitle');
```

Comment:
```suggestion
          // if `data-title` is present without a value, fall back to the title attr
          title = $this.attr('title') || $this.data('qtip-oldtitle');
```

This should be using 2-space indentation!